src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard.test.ts

```diff
@@ -0,0 +1,157 @@
+jest.mock('src/libs/ajax')
+
+describe('transforming user info to the request object', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it('splits lists of emails and maps them to roles', () => {
+    const emailList = 'a@b.com, b@c.com'
+    const result = userInfoListToProjectAccessObjects(emailList, 'User')
+
+    expect(result).toHaveLength(2)
+    expect(result[0].role).toEqual('User')
+    expect(result[0].email).toEqual('a@b.com')
+    expect(result[1].role).toEqual('User')
+    expect(result[1].email).toEqual('b@c.com')
+
+    // Empty emails returns an empty array
+    expect(userInfoListToProjectAccessObjects('', 'User')).toHaveLength(0)
+  })
+})
+
+const testStepActive = stepNumber => {
+  screen.queryAllByRole('listitem').forEach((step, index) => {
+    if (index === stepNumber - 1) {
+      expect(step.getAttribute('aria-current')).toBe('step')
+    } else {
+      expect(step.getAttribute('aria-current')).toBe('false')
+    }
+  })
+}
+
+describe('AzureBillingProjectWizard', () => {
+  let renderResult
+  const onSuccess = jest.fn()
+  const captureEvent = jest.fn()
+
+  beforeEach(() => {
+    renderResult = render(h(AzureBillingProjectWizard, {
+      onSuccess
+    }))
+  })
+
+  it('should not fail any accessibility tests in initial state', async () => {
+    expect(await axe(renderResult.container)).toHaveNoViolations()
+  })
+
+  it('should support happy path of submitting with no users/owners', async () => {
+    // Integration test of steps (act/arrange/assert not really possible)
+    const createAzureProject = jest.fn().mockResolvedValue({})
+    const billingProjectName = 'LotsOfCash'
+
+    testStepActive(1)
+    const managedApp = await selectManagedApp(captureEvent, createAzureProject)
+
+    testStepActive(2)
+    fireEvent.click(getNoUsersRadio())
+
+    testStepActive(3)
+    verifyCreateBillingProjectDisabled()
+    await nameBillingProject(billingProjectName)
+    expect(captureEvent).toBeCalledWith(Events.billingAzureCreationProjectNameEntered)
+    expect(await axe(renderResult.container)).toHaveNoViolations()
+
+    await clickCreateBillingProject()
+    expect(createAzureProject).toBeCalledWith(
+      billingProjectName, managedApp.tenantId, managedApp.subscriptionId, managedApp.managedResourceGroupId,
+      []
+    )
+    expect(onSuccess).toBeCalledWith(billingProjectName)
+  })
+
+  it('should support happy path of submitting with owners and users', async () => {
+    // Integration test of steps (act/arrange/assert not really possible)
+    const createAzureProject = jest.fn().mockResolvedValue({ ok: true })
+    const billingProjectName = 'LotsOfCashForAll'
+
+    testStepActive(1)
+    const managedApp = await selectManagedApp(captureEvent, createAzureProject)
+
+    testStepActive(2)
+    fireEvent.click(getAddUsersRadio())
+    addUserAndOwner('onlyuser@example.com', 'owner@example.com')
+    testStepActive(2) // Have to click in billing project input to become the active step
```

Comment:
Lou and I discussed this at length-- when you supply email addresses, there is no way to know that the user is "done". Therefore they must move focus to the billing project input to mark the next step as active (and they can't have any email errors).